pixart.js

```diff
@@ -1 +1,30 @@
+$(document).ready(function () {
 
+function changeColor(color) {
+  $(".brush").css("background-color",color)
+}
+
+$("#set-color")[0].addEventListener("click", function(event){
```

Comment:
Technically, this works, but it's a bit weird to mix jQuery and vanilla JS on the same line... better here would be:

```js
$("#set-color").on("click", function(event){
  // code here
})
```